caravel/assets/javascripts/explorev2/stores/store.js

```diff
@@ -51,3 +64,1618 @@ export const defaultOpts = {
+                 'relative to the time granularity selected',
+  },
+
+  series: {
+    type: 'SelectField',
+    label: 'Series',
+    choices: formatSelectOptions(['', 'series']),
+    default: '',
+    description: 'Defines the grouping of entities. ' +
+                 'Each series is shown as a specific color on the chart and ' +
+                 'has a legend toggle',
+  },
+
+  entity: {
+    type: 'SelectField',
+    label: 'Entity',
+    choices: formatSelectOptions(['', 'entity']),
+    default: '',
+    description: 'This define the element to be plotted on the chart',
+  },
+
+  x: {
+    type: 'SelectField',
+    label: 'X Axis',
+    choices: formatSelectOptions(['', 'metrics assigned to x']),
+    default: '',
+    description: 'Metric assigned to the [X] axis',
+  },
+
+  y: {
+    type: 'SelectField',
+    label: 'Y Axis',
+    choices: formatSelectOptions(['', 'metrics assigned to x']),
+    default: '',
+    description: 'Metric assigned to the [Y] axis',
+  },
+
+  size: {
+    type: 'SelectField',
+    label: 'Bubble Size',
+    default: '',
+    choices: formatSelectOptions(['', 'bubble-size']),
+  },
+
+  url: {
+    type: 'TextField',
+    label: 'URL',
+    description: 'The URL, this field is templated, so you can integrate ' +
+                 '{{ width }} and/or {{ height }} in your URL string.',
+    default: 'https: //www.youtube.com/embed/JkI5rg_VcQ4',
+  },
+
+  x_axis_label: {
+    type: 'TextField',
+    label: 'X Axis Label',
+    default: '',
+  },
+
+  y_axis_label: {
+    type: 'TextField',
+    label: 'Y Axis Label',
+    default: '',
+  },
+
+  where: {
+    type: 'TextField',
+    label: 'Custom WHERE clause',
+    default: '',
+    description: 'The text in this box gets included in your query\'s WHERE ' +
+                 'clause, as an AND to other criteria. You can include ' +
+                 'complex expression, parenthesis and anything else ' +
+                 'supported by the backend it is directed towards.',
+  },
+
+  having: {
+    type: 'TextField',
+    label: 'Custom HAVING clause',
+    default: '',
+    description: 'The text in this box gets included in your query\'s HAVING ' +
+                 'clause, as an AND to other criteria. You can include ' +
+                 'complex expression, parenthesis and anything else ' +
+                 'supported by the backend it is directed towards.',
+  },
+
+  compare_lag: {
+    type: 'TextField',
+    label: 'Comparison Period Lag',
+    description: 'Based on granularity, number of time periods to compare against',
+  },
+
+  compare_suffix: {
+    type: 'TextField',
+    label: 'Comparison suffix',
+    description: 'Suffix to apply after the percentage display',
+  },
+
+  table_timestamp_format: {
+    type: 'SelectField',
+    label: 'Table Timestamp Format',
+    default: 'smart_date',
+    choices: formatSelectOptions(TIME_STAMP_OPTIONS),
+    description: 'Timestamp Format',
+  },
+
+  series_height: {
```

Comment:
i think it should work like the current SelectCustomMultiField. we should be able to do that with this http://jedwatson.github.io/react-select/